aion/test/lib/ranking_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Aion.RankingTest do
+  use Aion.ModelCase
+  alias Aion.{Repo, User, UserCategoryScore, Ranking}
+
+  describe "Ranking.general" do
+    test "returns scored points in each category for each user" do
+      user1 = Repo.insert! User.registration_changeset(%User{}, %{name: "John", email: "test1@example.com", password: "test123"})
+      user2 = Repo.insert! User.registration_changeset(%User{}, %{name: "Michael", email: "test2@example.com", password: "test123"})
+      Repo.insert! %UserCategoryScore{user: user1, score: 2}
```

Comment:
1. How about defining those constants as macros? This is the approach we generally use in most of the tests
2. Consider using [insert_all](https://hexdocs.pm/ecto/Ecto.Repo.html#c:insert_all/3)